index.html

```diff
@@ -245,8 +245,8 @@ <h2 id="display-constraints">
           New Constraints for Captured Display Surfaces
         </h2>
         <p>
-          Two constraints are defined that allow an application to observe properties of the
-          selected <a>display surface</a>. Since the source of media cannot be changed after a
+          Following new constraints are defined that allow an application to observe the properties 
+          of the selected <a>display surface</a>. Since the source of media cannot be changed after a
```

Comment:
This sentence seems wrong - we do want the app to change the constraints, not just observe them.

Reply:
Good catch.

@suhasHere, we need to reframe this section.  Rather than do the obvious, and split the section on constraints into two, we should make it clear that though constraints don't affect selection (as much as people might like them to), they still affect the shape of the stream that is produced.

Reply:
@juberti @martinthomson I was planning to do constraints related work in a separate PR that also deals with bringing back few constraints as decided in the last meeting. Would it be ok, if I address the general constraints ideas in there ?